RescueRS/Presenter/Controllers/App/V1/Rescue/DTOs/RescueDTO.cs

```diff
@@ -12,12 +13,15 @@ public class RescueDTO
     public int AnimalsNumber { get; set; }
     public double Latitude { get; set; }
     public double Longitude { get; set; }
-    public bool Rescued { get; set; }
+    public RescueStatusEnum Status { get; set; }
     public DateTimeOffset? RescueDateTime { get; set; }
+    public DateTimeOffset? UpdateDateTime { get; set; }
     public string Cellphone { get; set; } = null!;
-    public double? distance { get; set; }
+    public string? Description { get; set; }
+    public double? Distance { get; set; }
+    public bool StartedByMe { get; set; }
```

Comment:
This is a technical debth. We should change the name of this attribute in the future, since this will be true if the user who requested the rescue cancel it